subsys/logging/log_output_syst.c

```diff
@@ -672,17 +673,84 @@ void log_output_string_syst_process(const struct log_output *log_output,
 
 	MIPI_SYST_PRINTF(&log_syst_handle, severity, str);
 }
+#else
+static void hexdump2_print(struct log_msg2 *msg,
+			  const struct log_output *output)
+{
+	char *buf;
+	size_t length;
+	uint32_t severity = level_to_syst_severity(log_msg2_get_level(msg));
 
-void log_output_hexdump_syst_process(const struct log_output *log_output,
-				struct log_msg_ids src_level,
-				const uint8_t *data, uint32_t length, uint32_t flag)
+	buf = log_msg2_get_data(msg, &length);
+	MIPI_SYST_WRITE(&log_syst_handle, severity, 0x1A, buf, length);
```

Comment:
It is unusual to see a hard coded value that is neither 0 nor 1.
1. What is the purpose of 0x1A?
2. If it is correct, can we use a descriptive macro?

Reply:
@peter-mitsis  just following the semantics of MIPI_SYS_WRITE used for v1 logging implementation for now.